### PR TITLE
Contact Me page

### PR DESCRIPTION
**Pull request by gaks1**
What's done:
1. Add html.
2. Add style.
3. Linters checked.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -1381,6 +1383,226 @@
...
+    color: #fff;
+  }
+
+  .form-button:disabled {
+    border: 1px solid #c1c7d0;
+    color: #5e6c84;
+  }
 }
 
 @media only screen and (min-width: 375px) {
```
Comment (MaganaAsiati) [inline style.css]: - Kindly I suggest you use the  appropriate screen size  I think that is very small and your page is not responsive in the mobile version I suggest you use for example `min-width:768px`


image

**Review by MaganaAsiati — CHANGES_REQUESTED** [review]
# STATUS: Required Changes ♻️♻️♻️ 
## Hi @gaks1 , I'm @MaganaAsiati, your reviewer for this PR  :wave: 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

so-close-gif-4

## Highlights

- Descriptive README file  :heavy_check_mark:👨🏻‍💻
- Great work using the right GitHub Flow.:heavy_check_mark:
- No Linter errors :heavy_check_mark:

## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._
- [ ] [OPTIONAL] Please make sure to make your commit messages more descriptive in the future. :heavy_check_mark:👨🏻‍💻
- [ ] [OPTIONAL] kindly  Add a meaningful description to your pull request in the future which gives more information about what you implement in this milestone :heavy_check_mark:👨🏻‍💻


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: style.css (hunk 1 above)
Comment (gaks1) [reply style.css]: Hello, In the Figma design, the mobile design requirements are at a width of 375px. My previous code reviewers also recommended using the design width.  So should still change it?

Comment (gaks1) [reply style.css]: @MaganaAsiati

Comment (Shedrack-Sunday) [reply style.css]: Hello @gaks1 , That's right. the mobile version should be `375px`. I'll review your code and end my suggestions. Please check them below

File: main.html [hunk 2]
```diff
@@ -284,6 +284,23 @@ <h2 class="about-header">About Myself</h2>
...
+                <ul class="con-f-list">
+                    <li><input class="name" type="text" name="username"  placeholder="Yeremias" maxlength="30" required></li>
+                    <li><input class="email"  type="email" name="useremail"  placeholder="natajayanj14@gmail.com" required></li>
+                    <li><textarea class="feedback" name="userfeedback" placeholder="Write your message here" cols="30" rows="10" maxlength="500"
+                            required></textarea></li>
+                    <li><button class="form-button">Get It Touch</button></li>
+                </ul>
+            </form>
+        
+        </section>
```
Comment (Shedrack-Sunday) [inline main.html:302]: - [ ] The previous reviewer rightly pointed out the need to make it responsive on the mobile view. you did a good job with the design, but it's fixed I've attached a screenshot to show you how it looks, and the equivalent on figma. kindly adjust your CSS.

# Your design :
image

# Figma :
image

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hi @gaks1    👋 

# CHANGES REQUIRED ♻️

Great work on the project 👏
You've done well implementing most of the features from the project requirement, but there are still some which aren't been addressed yet which ha been pointed by your previous reviewer . 



Highlights
✅All linters are checked
✅ Descriptive commit messages 

Required Changes ♻️


- [ ] The main changes needed have to do with your styling. the media queries you used seem to have made the deign static. find a more detail review in the comment section  

_check the comment section for a detailed review_

Optional suggestions
Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.


Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification.


Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: main.html (hunk 2 above)
Comment (gaks1) [reply main.html:302]: @Shedrack-Sunday  So the width should still be 375px.

Comment (gaks1) [reply main.html:302]: Should the responsiveness be for 375 px and below @Shedrack-Sunday

Comment (Shedrack-Sunday) [reply main.html:302]: Yes, please. Sorry for the late reply

**Review by jerryowusu — APPROVED** [review]
Hi @gaks1 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

chelsea-mason-approve


## Highlights
Form submitted successfully :heavy_check_mark: 
Great effort in making the requested changes :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**